docs/source/notebooks/opendap.ipynb

```diff
@@ -4,13 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# OPeNDAP\n",
```

Comment:
Since you are in this notebook, can you take a look at the `IndexError: too many indices for array` error, see http://jenkins.ouranos.ca/job/PAVICS-e2e-workflow-tests/job/master/85/console

Reply:
Fixed.